Makefile

```diff
@@ -31,3 +31,14 @@ $(call add-bindata,v3.11.0,./bindata/v3.11.0/...,bindata,v311_00_assets,pkg/oper
 clean:
 	$(RM) ./cluster-config-operator
 .PHONY: clean
+
+update-codegen:
```

Comment:
please rename the target to `update-codegen-crd` and you can hook it into `update-codegen: update-codegen-crd` if you want so it doesn't conflict with standart library-go targets